unison-src/new-runtime-transcripts/utf8.md

```diff
@@ -36,7 +37,7 @@ We can check that encoding and then decoding should give us back the same `Text`
 checkRoundTrip: Text -> [Result]
 checkRoundTrip t = 
   bytes = toUtf8 t
-  match fromUtf8 bytes with 
+  match Text.fromUtf8 bytes with 
```

Comment:
Just wondering if this change was needed for some reason... it shouldn't be I don't think, unless the old `fromUtf8` is still in the starting namespace.